steps/ascii-generator/1.0.0/step.yml

```diff
@@ -0,0 +1,52 @@
+title: ASCII art generator
+summary: |
+  Add a random Bitrise ASCII art to your build log
+description: |
+  Add a random Bitrise ASCII art to your build log
+website: https://github.com/slapec93/bitrise-step-ascii-art-generator
+source_code_url: https://github.com/slapec93/bitrise-step-ascii-art-generator
+support_url: https://github.com/slapec93/bitrise-step-ascii-art-generator/issues
+published_at: 2018-12-12T10:28:07.521638199+01:00
+source:
+  git: https://github.com/bitrise-steplib/bitrise-step-ascii-art-generator.git
+  commit: 2a5fd25b98fcff9e9d8e7c735c8a46da232b459b
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- utility
+toolkit:
+  go:
+    package_name: github.com/slapec93/steps-ascii-art-generator
+deps:
+  brew:
+  - name: git
```

Comment:
does this step use `git` and `wget`? these dependencies will be installed before the step run, which increase the step runtime